src/controller/CHIPDeviceController.h

```diff
@@ -141,6 +141,13 @@ struct ControllerInitParams
      */
     bool removeFromFabricTableOnShutdown = true;
 
+    /**
+     * Specifies whether to utilize a storage fabric for initializing the DeviceController.
```

Comment:
What is a "storage fabric index". What is desired here seems like it can be done by calling existing APIs with an externally managed PersistentStorageDelegate that always starts empty. Adding this adds complexity that is not clear it is needed.

Reply:
Yes, by configuring FactoryInitParams with 'fabricIndependentStorage', 'operationalKeystore', and 'opCertStore', we enable the use of a storage fabric. However, member 'mFabricIndex' in the DeviceController class cannot be updated unless the NOC chain and operational key pair are provided.